style/scss/_details.scss

```diff
@@ -0,0 +1,476 @@
+.main {
+  box-shadow: 0px 1px 18px 0px rgba($dark-gray, 0.35);
+  margin: 0 20px;
+  border-radius: 30px;
+  padding-bottom: 30px;
+}
+
+.back__wrapper {
+  margin: 140px 20px 20px 20px;

```

Comment:
Please use values in REMs (check _reset.scss, for our project 1REM = 10px).

html {
  font-size: 10px;
}